# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../net-tools/net-tools-0001-remove-obsolete-commands.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From f9c5e446972d81873c1c6b614584bd6b293a1242 Mon Sep 17 00:00:00 2001
From: Christian Wiese <chris@opensde.org>
Date: Wed, 19 Dec 2012 13:05:50 +0100
Subject: [PATCH] remove obsolete commands

hostname is provided by inetutils
{nis,yp}domainname is provided by yp-tools
---
 Makefile                  |   14 +---
 man/de_DE/dnsdomainname.1 |    1 -
 man/de_DE/domainname.1    |    1 -
 man/de_DE/hostname.1      |  229 ---------------------------------------------
 man/de_DE/nisdomainname.1 |    1 -
 man/de_DE/ypdomainname.1  |    1 -
 man/en_US/dnsdomainname.1 |    1 -
 man/en_US/domainname.1    |    1 -
 man/en_US/hostname.1      |  213 -----------------------------------------
 man/en_US/nisdomainname.1 |    1 -
 man/en_US/ypdomainname.1  |    1 -
 man/fr_FR/dnsdomainname.1 |    1 -
 man/fr_FR/domainname.1    |    1 -
 man/fr_FR/hostname.1      |  212 -----------------------------------------
 man/fr_FR/nisdomainname.1 |    1 -
 man/fr_FR/ypdomainname.1  |    1 -
 man/pt_BR/dnsdomainname.1 |    1 -
 man/pt_BR/domainname.1    |    1 -
 man/pt_BR/hostname.1      |  191 -------------------------------------
 man/pt_BR/nisdomainname.1 |    1 -
 man/pt_BR/ypdomainname.1  |    1 -
 21 files changed, 2 insertions(+), 873 deletions(-)
 delete mode 100644 man/de_DE/dnsdomainname.1
 delete mode 100644 man/de_DE/domainname.1
 delete mode 100644 man/de_DE/hostname.1
 delete mode 100644 man/de_DE/nisdomainname.1
 delete mode 100644 man/de_DE/ypdomainname.1
 delete mode 100644 man/en_US/dnsdomainname.1
 delete mode 100644 man/en_US/domainname.1
 delete mode 100644 man/en_US/hostname.1
 delete mode 100644 man/en_US/nisdomainname.1
 delete mode 100644 man/en_US/ypdomainname.1
 delete mode 100644 man/fr_FR/dnsdomainname.1
 delete mode 100644 man/fr_FR/domainname.1
 delete mode 100644 man/fr_FR/hostname.1
 delete mode 100644 man/fr_FR/nisdomainname.1
 delete mode 100644 man/fr_FR/ypdomainname.1
 delete mode 100644 man/pt_BR/dnsdomainname.1
 delete mode 100644 man/pt_BR/domainname.1
 delete mode 100644 man/pt_BR/hostname.1
 delete mode 100644 man/pt_BR/nisdomainname.1
 delete mode 100644 man/pt_BR/ypdomainname.1

diff --git a/Makefile b/Makefile
index cdea348..e41d727 100644
--- a/Makefile
+++ b/Makefile
@@ -31,7 +31,7 @@ BASEDIR ?= $(DESTDIR)
 NET_LIB_PATH = lib
 NET_LIB_NAME = net-tools
 
-PROGS	:= ifconfig hostname arp netstat route rarp slattach plipconfig nameif
+PROGS	:= ifconfig arp netstat route rarp slattach plipconfig nameif
 
 -include config.make
 ifeq ($(HAVE_IP_TOOLS),1)
@@ -197,7 +197,6 @@ installbin:
 	install -m 0755 -d ${BASEDIR}/sbin
 	install -m 0755 -d ${BASEDIR}/bin
 	install -m 0755 arp        ${BASEDIR}/sbin
-	install -m 0755 hostname   ${BASEDIR}/bin
 	install -m 0755 ifconfig   ${BASEDIR}/bin
 	install -m 0755 nameif     ${BASEDIR}/sbin
 	install -m 0755 netstat    ${BASEDIR}/bin
@@ -212,21 +211,12 @@ endif
 ifeq ($(HAVE_MII),1)
 	install -m 0755 mii-tool   $(BASEDIR)/sbin
 endif
-	ln -fs hostname $(BASEDIR)/bin/dnsdomainname
-	ln -fs hostname $(BASEDIR)/bin/ypdomainname
-	ln -fs hostname $(BASEDIR)/bin/nisdomainname
-	ln -fs hostname $(BASEDIR)/bin/domainname
-ifeq ($(HAVE_AFDECnet),1)
-	ln -fs hostname $(BASEDIR)/bin/nodename
-endif
 
 savebin:
 	@for i in ${BASEDIR}/sbin/arp ${BASEDIR}/sbin/ifconfig \
                  ${BASEDIR}/bin/netstat \
 		 ${BASEDIR}/sbin/rarp ${BASEDIR}/sbin/route \
-		 ${BASEDIR}/bin/hostname ${BASEDIR}/bin/ypdomainname \
-                 ${BASEDIR}/bin/dnsdomainname ${BASEDIR}/bin/nisdomainname \
-		 ${BASEDIR}/bin/domainname ; do \
+		 ; do \
 		 [ -f $$i ] && cp -f $$i $$i.old ; done ; echo Saved.
 
 installdata:
diff --git a/man/de_DE/dnsdomainname.1 b/man/de_DE/dnsdomainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/de_DE/dnsdomainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
diff --git a/man/de_DE/domainname.1 b/man/de_DE/domainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/de_DE/domainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
diff --git a/man/de_DE/hostname.1 b/man/de_DE/hostname.1
deleted file mode 100644
index ce0cadb..0000000
--- a/man/de_DE/hostname.1
+++ /dev/null
@@ -1,229 +0,0 @@
-.\"
-.\" Original by ??
-.\"
-.\" German translation by Ralf Baechle (ralf@gnu.org)
-.\"
-.TH HOSTNAME 1 "6. M\(:arz 1999" "net-tools" "Handbuch f\(:ur Linuxprogrammierer"
-
-.SH NAME
-hostname \- den Rechnernamen anzeigen oder setzen.
-.br
-domainname \- den NIS/YP Domainnamen anzeigen oder setzen
-.br
-dnsdomainname \- den DNS Domainnamen des Systems anzeigen
-.br
-nisdomainname \- den NIS/YP Domainnamen anzeigen oder setzen
-.br
-ypdomainname \- den NIS/YP Domainnamen anzeigen oder setzen
-.br
-nodename \- den DECnet-Knotennamen anzeigen oder setzen
-
-.SH SYNOPSIS
-.B hostname
-.RB [ \-v ]
-.RB [ \-a ]
-.RB [ \-\-alias ]
-.RB [ \-d ]
-.RB [ \-\-domain ]
-.RB [ \-f ]
-.RB [ \-\-fqdn ]
-.RB [ \-i ]
-.RB [ \-\-ip-address ]
-.RB [ \-\-long ]
-.RB [ \-s ]
-.RB [ \-\-short ]
-.RB [ \-y ]
-.RB [ \-\-yp ]
-.RB [ \-\-nis ]
-.RB [ \-n ]
-.RB [ \-\-node ]
-
-.PP
-.B hostname 
-.RB [ \-v ]
-.RB [ \-F\ Dateiname ]
-.RB [ \-\-file\ Dateiname ]
-.RB [ Rechnername ]
-
-.PP
-.B domainname
-.RB [ \-v ]
-.RB [ \-F\ Dateiname ]
-.RB [ \-\-file\ Dateiname ]
-.RB [ Name ]
-
-.PP 
-.B nodename
-.RB [ \-v ]
-.RB [ \-F\ Dateiname ]
-.RB [ \-\-file\ Dateiname ]
-.RB [ Name ]
-
-.PP
-.B hostname
-.RB [ \-v ]
-.RB [ \-h ]
-.RB [ \-\-help ]
-.RB [ \-V ]
-.RB [ \-\-version ]
-
-.PP
-.B dnsdomainname
-.RB [ \-v ]
-.br
-.B nisdomainname
-.RB [ \-v ]
-.br
-.B ypdomainname
-.RB [ \-v ]
-
-.SH BESCHREIBUNG
-.B Hostname
-ist das Programm, das zum Anzeigen oder setzen des aktuellen Host-, Domain-,
-oder Knotennamens des Systems verwendet wird.  Diese Namen werden von vielen
-der Netzwerksprogrammen zur Identifikation der Maschine verwendet.
-Au\(sserdem wird der Domainname von NIS/YP verwendet.
-
-.SS "NAMEN AUSLESEN"
-Wenn ohne Argumente augerufen, zeigen die Programme den aktuellen Namen an:
-
-.LP
-.B hostname
-zeigt den aktuellen Rechnernamen des Systems, wie von der 
-.BR gethostname (2)
-Funktion ermittelt wird, an.
-
-.LP
-.B "domainname, nisdomainname, ypdomainname"
-zeigen den Namen des Systems, wie er von der
-.BR getdomainname (2)
-Funktion ermittelt wird, an.  Dieser Name wird auch der YP/NIS Domainname
-des Systems genannt.
-
-.LP
-.B nodename
-druckt den DECnet-Knotennamen des Systems, wie er von der
-.BR getnodename (2)
-Function ermittelt wird, aus.
-
-.LP
-.B dnsdomainname
-druckt den Domainteil des FQDN (Fully Qualified Domain Name oder zu deutsch
-vollst\(:andig spezifizierter Domainname) aus.  Der vollst\(:andige FQDN
-des Systems wird von
-.B "hostname \-\-fqdn"
-ausgegeben.
-
-.SS "NAME SETZEN"
-Wenn mit nur einem Argument oder mit der
-.B \-\-file
-Option aufgerufen, dann setzen diese Kommandos den Rechnernamen, den NIS/YP
-Domainnamen oder den DECnet-Knotennamen.
-
-.LP
-Nur der Superuser darf den Namen \(:andern.
-
-.LP
-Es ist nicht m\(:ogich den FQDN oder den DNS Domainnamen mit dem
-.B dnsdomainname 
-Befehl (Siehe 
-.B "Der FQDN"
-unten) zu setzen.
-
-.LP
-Der Rechnername wird \(:ublicherweise einmal beim Systemstart in
-.I /etc/rc.d/rc.inet1
-or
-.I /etc/init.d/boot
-gesetzt.  Der Rechnername wird dabei \(:ublicherweise aus einer Datei
-die den Rechnernamen, z.B.
-.I /etc/HOSTNAME
-gesetzt.
-
-.SS DER FQDN
-Der FQDN (wie er von
-.B "hostname \-\-fqdn"
-ausgegeben wird) 
-oder der DNS-Domainname (wie er von
-.B "dnsdomainname"
-ausgegeben wird) 
-kann nicht mit diesem Befehl ge\(:andert werden.  Der FQDN eines Systems ist
-der Name, den der
-.BR resolver (3)
-als den Namen des Systems zur\(:uckgibt.
-
-.LP
-Technisch: Der FQDN ist der Name, den
-.BR gethostbyname (2)
-f\(:ur den Rechnernamen zur\(:uckgibt, den
-.B gethostname (2)
-zur\(:uckgibt.
-Der DNS-Domainname ist der Teil nach dem ersten Punkt.
-.LP
-Deshalb h\(:angt es \(:ublicherweise von der Konfiguration in
-.IR /etc/host.conf )
-wie der Name ge\(:andert werden kann.  \(:Ublicherweise (wenn die Hosts Datei
-gelesen wird, bevor auf DNS oder NIS/YP zugegriffen wird) kann er in
-.I /etc/hosts
-ge\(:andert werden.
-
-
-.SH OPTIONEN
-.TP
-.I "\-a, \-\-alias"
-Den Aliasnamen des Rechners ausgeben, falls benutzt.
-.TP
-.I "\-d, \-\-domain"
-Den Namen der DNS-Domain ausgeben.  Der Befehl
-.B domainname
-kann nicht dazu verwendet werden um den Namen der DNS-Dom\(:ane auszugeben,
-da er den NIS Domainname und nicht den DNS Domainnamen ausgibt.  Zu diesem
-Zweck dient der Befehl
-.BR dnsdomainname .
-.TP
-.I "\-F, \-\-file filename"
-Den Rechnernamen aus der angegebenen Datei lesen.   Zeilen, die mit einem
-#-Zeichen anfangen sind Kommentare und werden ignoriert.
-.TP
-.I "\-f, \-\-fqdn, \-\-long"
-Den FQDN (Fully Qualified Domain Name) anzeigen.  Ein FQDN besteht aus einem
-kurzen Rechnernamen ohne Punkt und dem DNS-Domainnamen.  Au\(sser wenn BIND
-oder NIS zum Aufl\(:osen von Rechnernamen verwendet wird, kann der FQDN und
-der DNS-Domainname, der Teil des FQDN ist, in der Datei \fI/etc/hosts\fR
-ge\(:andert werden.
-.TP
-.I "\-h, \-\-help"
-Kurzanleitung ausdrucken und beenden.
-.TP
-.I "\-i, \-\-ip-address"
-Die IP-Adresse(n) des Rechners anzeigen und beenden.
-.TP
-.I "\-n, \-\-node"
-Den DECnet-Knotennamen anzeigen.  Wenn ein Argument oder die
-.B \-\-file name
-Option angegeben wird, dann kann Root auch einen neuen Knotennamen setzen.
-.TP
-.I "\-s, \-\-short"
-Den Kurznamen anzeigen.  Dies ist der ab dem ersten Punkt abgeschnittene
-Rechnername.
-.TP
-.I "\-V, \-\-version"
-Versionsinformation auf der Standardausgabe ausgeben und erfolgreich beenden.
-.TP
-.I "\-v, \-\-verbose"
-Ausf\(:uhrlichere Ausgaben.
-.TP
-.I "\-y, \-\-yp, \-\-nis"
-Den NIS-Domainnamen anzeigen.  Wenn ein Argument oder die 
-.B \-\-file name
-Option gegeben wird, dann kann Root auch eine neue NIS-Domain setzen.
-.SH DATEIEN
-.B /etc/hosts
-.SH AUTOREN
-Peter Tobias, <tobias@et-inf.fho-emden.de>
-.br
-Bernd Eckenfels, <net-tools@lina.inka.de> (NIS and manpage).
-.br
-Steve Whitehouse, <SteveW@ACM.org> (DECnet support and manpage).
-.SH \(:Ubersetzung
-Ralf B\(:achle <ralf@gnu.org>
diff --git a/man/de_DE/nisdomainname.1 b/man/de_DE/nisdomainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/de_DE/nisdomainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
diff --git a/man/de_DE/ypdomainname.1 b/man/de_DE/ypdomainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/de_DE/ypdomainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
diff --git a/man/en_US/dnsdomainname.1 b/man/en_US/dnsdomainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/en_US/dnsdomainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
diff --git a/man/en_US/domainname.1 b/man/en_US/domainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/en_US/domainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
diff --git a/man/en_US/hostname.1 b/man/en_US/hostname.1
deleted file mode 100644
index 9f7dd7f..0000000
--- a/man/en_US/hostname.1
+++ /dev/null
@@ -1,213 +0,0 @@
-.TH HOSTNAME 1 "2008\-10\-03" "net\-tools" "Linux System Administrator's Manual"
-
-.SH NAME
-hostname \- show or set the system's host name
-.br
-domainname \- show or set the system's NIS/YP domain name
-.br
-dnsdomainname \- show the system's DNS domain name
-.br
-nisdomainname \- show or set system's NIS/YP domain name
-.br
-ypdomainname \- show or set the system's NIS/YP domain name
-.br
-nodename \- show or set the system's DECnet node name
-
-.SH SYNOPSIS
-.B hostname
-.RB [ \-v ]
-.RB [ \-a ]
-.RB [ \-\-alias ]
-.RB [ \-d ]
-.RB [ \-\-domain ]
-.RB [ \-f ]
-.RB [ \-\-fqdn ]
-.RB [ \-i ]
-.RB [ \-\-ip\-address ]
-.RB [ \-\-long ]
-.RB [ \-s ]
-.RB [ \-\-short ]
-.RB [ \-y ]
-.RB [ \-\-yp ]
-.RB [ \-\-nis ]
-.RB [ \-n ]
-.RB [ \-\-node ]
-
-.PP
-.B hostname 
-.RB [ \-v ]
-.RB [ \-F\ filename ]
-.RB [ \-\-file\ filename ]
-.RB [ hostname ]
-
-.PP
-.B domainname
-.RB [ \-v ]
-.RB [ \-F\ filename ]
-.RB [ \-\-file\ filename ]
-.RB [ name ]
-
-.PP 
-.B nodename
-.RB [ \-v ]
-.RB [ \-F\ filename ]
-.RB [ \-\-file\ filename ]
-.RB [ name ]
-
-.PP
-.B hostname
-.RB [ \-v ]
-.RB [ \-h ]
-.RB [ \-\-help ]
-.RB [ \-V ]
-.RB [ \-\-version ]
-
-.PP
-.B dnsdomainname
-.RB [ \-v ]
-.br
-.B nisdomainname
-.RB [ \-v ]
-.br
-.B ypdomainname
-.RB [ \-v ]
-
-.SH DESCRIPTION
-.B Hostname
-is the program that is used to either set or display
-the current host, domain or node name of the system.  These names are used
-by many of the networking programs to identify the machine. The domain 
-name is also used by NIS/YP.
-
-.SS "GET NAME"
-When called without any arguments, the program displays the current
-names:
-
-.LP
-.B hostname
-will print the name of the system as returned by the
-.BR gethostname (2)
-function.
-
-.LP
-.B "domainname, nisdomainname, ypdomainname"
-will print the name of the system as returned by the
-.BR getdomainname (2)
-function. This is also known as the YP/NIS domain name of the system.
-
-.LP
-.B nodename
-will print the DECnet node name of the system as returned by the
-.BR getnodename (2)
-function.
-
-.LP
-.B dnsdomainname
-will print the domain part of the FQDN (Fully Qualified Domain Name). The
-complete FQDN of the system is returned with
-.BR "hostname \-\-fqdn" .
-
-.SS "SET NAME"
-When called with one argument or with the 
-.B \-\-file
-option, the commands set the host name, the NIS/YP domain name or
-the node name. 
-
-.LP
-Note, that only the super-user can change the names. 
-
-.LP
-It is not possible to set the FQDN or the DNS domain name with the 
-.B dnsdomainname 
-command (see 
-.B "THE FQDN"
-below).
-
-.LP
-The host name is usually set once at system startup in
-.I /etc/rc.d/rc.inet1
-or
-.I /etc/init.d/boot
-(normally by reading the contents of a file which contains
-the host name, e.g. 
-.IR /etc/hostname ).
-
-.SS THE FQDN
-You can't change the FQDN (as returned by 
-.BR "hostname \-\-fqdn" ) 
-or the DNS domain name (as returned by
-.BR "dnsdomainname" ) 
-with this command. The FQDN of the system is the name that the
-.BR resolver (3)
-returns for the host name. 
-
-.LP
-Technically: The FQDN is the name
-.BR gethostbyname (2)
-returns for the host name returned by
-.BR gethostname (2).
-The DNS domain name is the part after the first dot.
-.LP
-Therefore it depends on the configuration (usually in 
-.IR /etc/host.conf )
-how you can change it. Usually (if the hosts file is parsed before DNS or 
-NIS) you can change it in 
-.IR /etc/hosts .
-
-
-.SH OPTIONS
-.TP
-.I "\-a, \-\-alias"
-Display the alias name of the host (if used).
-.TP
-.I "\-d, \-\-domain"
-Display the name of the DNS domain. Don't use the command
-.B domainname
-to get the DNS domain name because it will show the NIS domain name and
-not the DNS domain name. Use
-.B dnsdomainname 
-instead.
-.TP
-.I "\-F, \-\-file filename"
-Read the host name from the specified file. Comments (lines starting with
-a `#') are ignored.
-.TP
-.I "\-f, \-\-fqdn, \-\-long"
-Display the FQDN (Fully Qualified Domain Name). A FQDN consists of a
-short host name and the DNS domain name. Unless you are using bind or NIS
-for host lookups you can change the FQDN and the DNS domain name (which is
-part of the FQDN) in the \fI/etc/hosts\fR file.
-.TP
-.I "\-h, \-\-help"
-Print a usage message and exit.
-.TP
-.I "\-i, \-\-ip\-address"
-Display the IP address(es) of the host.
-.TP
-.I "\-n, \-\-node"
-Display the DECnet node name. If a parameter is given (or
-.B \-\-file name
-) the root can also set a new node name.
-.TP
-.I "\-s, \-\-short"
-Display the short host name. This is the host name cut at the first dot.
-.TP
-.I "\-V, \-\-version"
-Print version information on standard output and exit successfully.
-.TP
-.I "\-v, \-\-verbose"
-Be verbose and tell what's going on.
-.TP
-.I "\-y, \-\-yp, \-\-nis"
-Display the NIS domain name. If a parameter is given (or 
-.B \-\-file name
-) then root can also set a new NIS domain.
-.SH FILES
-.B /etc/hosts
-.SH AUTHOR
-Peter Tobias, <tobias@et\-inf.fho\-emden.de>
-.br
-Bernd Eckenfels, <net\-tools@lina.inka.de> (NIS and manpage).
-.br
-Steve Whitehouse, <SteveW@ACM.org> (DECnet support and manpage).
-
diff --git a/man/en_US/nisdomainname.1 b/man/en_US/nisdomainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/en_US/nisdomainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
diff --git a/man/en_US/ypdomainname.1 b/man/en_US/ypdomainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/en_US/ypdomainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
diff --git a/man/fr_FR/dnsdomainname.1 b/man/fr_FR/dnsdomainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/fr_FR/dnsdomainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
diff --git a/man/fr_FR/domainname.1 b/man/fr_FR/domainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/fr_FR/domainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
diff --git a/man/fr_FR/hostname.1 b/man/fr_FR/hostname.1
deleted file mode 100644
index 3adaf4b..0000000
--- a/man/fr_FR/hostname.1
+++ /dev/null
@@ -1,212 +0,0 @@
-.TH HOSTNAME 1 "15 Mars 1999" "net-tools" "Linux Programmer's Manual"
-.SH NOM
-hostname \- affiche ou définit le nom d'hôte du système
-.br
-domainname \- affiche le nom de domaine NIS/YP du système
-.br
-dnsdomainname \- affiche le nom de domaine du système
-.br
-nisdomainname \- affiche ou définit le nom de domaine NIS/YP du système
-.br
-ypdomainname \- affiche ou définit le nom de domaine NIS/YP du système
-.br
-nodename \- affiche ou définit le nom de domaine DECnet du système
-
-.SH SYNOPSIS
-.B hostname
-.RB [ \-v ]
-.RB [ \-a ]
-.RB [ \-\-alias ]
-.RB [ \-d ]
-.RB [ \-\-domain ]
-.RB [ \-f ]
-.RB [ \-\-fqdn ]
-.RB [ \-i ]
-.RB [ \-\-ip-address ]
-.RB [ \-\-long ]
-.RB [ \-s ]
-.RB [ \-\-short ]
-.RB [ \-y ]
-.RB [ \-\-yp ]
-.RB [ \-\-nis ]
-.RB [ \-n ]
-.RB [ \-\-node ]
-
-.PP
-.B hostname 
-.RB [ \-v ]
-.RB [ \-F\ nom_fichier ]
-.RB [ \-\-file\ nom_fichier ]
-.RB [ hostname ]
-
-.PP
-.B domainname
-.RB [ \-v ]
-.RB [ \-F\ nom_fichier ]
-.RB [ \-\-file\ nom_fichier ]
-.RB [ name ]
-
-.PP 
-.B nodename
-.RB [ \-v ]
-.RB [ \-F\ nom_fichier ]
-.RB [ \-\-file\ nom_fichier ]
-.RB [ name ]
-
-.PP
-.B hostname
-.RB [ \-v ]
-.RB [ \-h ]
-.RB [ \-\-help ]
-.RB [ \-V ]
-.RB [ \-\-version ]
-
-.PP
-.B dnsdomainname
-.RB [ \-v ]
-.br
-.B nisdomainname
-.RB [ \-v ]
-.br
-.B ypdomainname
-.RB [ \-v ]
-
-.SH DESCRIPTION
-.B Hostname
-est le programme utilisé soit pour définir le nom d'hôte, soit
-pour visualiser le nom d'hôte ou de domaine du système. Ce nom est
-utilisé par différents programmes réseaux pour identifier la machine. Le
-nom de domaine est également utilisé par les NIS/YP.
-
-.SS "OBTENTION DU NOM"
-Appelé sans argument, le programme affiche les noms courants:
-
-.LP
-.B hostname
-fournit le nom du système tel qu'il est retourné par la fonction
-.BR getdomainname (2).
-Ceci est connu comme nom de domaine YP/NIS du système.
-
-
-.LP
-.B nodename
-fournit le nom de noeud DECnet du système tel qu'il est retourné par la
-fonction
-.BR getnodename (2)
-.
-
-.LP
-.B dnsdomainname
-fournit la partie domaine du nom complètement qualifié - FQDN (Fully
-Qualified Domain Name). Le nom FQDN du système est retourné par la commande
-.BR "hostname \-\-fqdn" .
-
-.SS "SET NAME"
-Appelé avec un argument ou avec l'option
-.B \-\-file
-, la commande définit le nom d'hôte, le nom de domaine NIS/YP ou
-le nom de noeud. 
-
-.LP
-Notez que seul le super-utilisateur peut changer les noms.
-
-.LP
-Il n'est pas possible de définir le nom FQDN ou le nom de domaine DNS avec
-la commande 
-.B dnsdomainname 
- (voir 
-.B "LE FQDN"
-ci-dessous).
-
-.LP
-Le nom d'hôte est générallement définit une seule fois, au
-démarrage du système, par le script de configuration
-.I /etc/rc.d/rc.inet1.
-ou
-.I /etc/init.d/boot
-(normallement en lisant un fichier contenant ce nom, p.ex. 
-.IR /etc/hostname ).
-
-.SS LE FQDN
-Vous pouvez modifier le FQDN (tel que fourni par la commande 
-.BR "hostname \-\-fqdn" ) 
-ou le nom de domaine DNS (tel que fourni par la commande
-.BR "dnsdomainname" ) 
-avec cette commande. Le FQDN du système est le nom que le resolveur (
-.BR resolver (3)
- ) retourne comme nom d'hôte. 
-
-.LP
-Techniquement: Le FQDN est le nom fourni par
-.BR gethostbyname (2)
-à partir du nom d'hôte retourné par
-.BR gethostname (2).
-Le nom de domaine DNS est la partie suivant le premier point.
-.LP
-La façon dont vous pouvez le changer dépend de la configuration
-(habituellement dans 
-.IR /etc/host.conf ).
-Si le fichier 'hosts' est analysé avant d'interroger le DNS ou 
-NIS) vous pouvez le changer dans le fichier 
-.IR /etc/hosts .
-
-.SH OPTIONS
-.TP
-.I "\-a, \-\-alias"
-Affiche l'alias de l'hôte (si utilisé).
-.TP
-.I "\-d, \-\-domain"
-Affiche le nom du domaine DNS. Ne pas utiliser la commande
-.B domainname
-pour obtenir le nom de domaine DNS car vous obtiendrez en réalité le
-nom de domaine NIS. Utilisez plutôt la commande
-.B dnsdomainname 
-.
-.TP
-.I "\-F, \-\-file nom_fichier"
-Lit le nom d'hôte à partir du fichier spécifié. Les commentaires (lignes commençant
-par un `#') sont ignorés.
-.TP
-.I "\-f, \-\-fqdn, \-\-long"
-Affiche le nom pleinement qualifié - FQDN (Fully Qualified Domain Name). un
-FQDN consiste en un nom d'hôte court et un nom de domaine. A moins que vous
-utilisiez 'bind' ou les NIS pour la recherche d'hôtes, vous pouvez changer
-le nom FQDN et le nom de domaine (qui fait partie du FQDN) dans le fichier
-\fI/etc/hosts\fR.
-.TP
-.I "\-h, \-\-help"
-Affiche un message d'aide et se termine.
-.TP
-.I "\-i, \-\-ip-address"
-Affiche la (les) adresse(s) IP de l'hôte.
-.TP
-.I "\-n, \-\-node"
-Affiche le nom de noeud DECnet. Si un paramètre est fourni (ou
-.B \-\-file nom_fichier
-) le super-utilisateur peut également définir un nouveau nom de neud.
-.TP
-.I "\-s, \-\-short"
-Affiche le nom d'hôte en format court. Il s'agit du nom d'hôte coupé au
-premier point.
-.TP
-.I "\-V, \-\-version"
-Affiche les informations sur la version sur la sortie standard et se
-termine correctement.
-.TP
-.I "\-v, \-\-verbose"
-Mode verbeux.
-.TP
-.I "\-y, \-\-yp, \-\-nis"
-Affiche le nom de domaine NIS. Si un paramètre est fourni (ou
-.B \-\-file nom_fichier
-) le super-utilisateur peut également définir un nouveau nom de domaine NIS.
-.SH FICHIERS
-.B /etc/hosts
-.SH AUTEURS
-Peter Tobias, <tobias@et-inf.fho-emden.de>
-.br
-Bernd Eckenfels, <net-tools@lina.inka.de> (NIS et pages de manuel).
-.br
-Steve Whitehouse, <SteveW@ACM.org> (Support DECnet et pages de manuel).
-.SH TRADUCTION
-Jean Michel VANSTEENE (vanstee@worldnet.fr)
diff --git a/man/fr_FR/nisdomainname.1 b/man/fr_FR/nisdomainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/fr_FR/nisdomainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
diff --git a/man/fr_FR/ypdomainname.1 b/man/fr_FR/ypdomainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/fr_FR/ypdomainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
diff --git a/man/pt_BR/dnsdomainname.1 b/man/pt_BR/dnsdomainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/pt_BR/dnsdomainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
diff --git a/man/pt_BR/domainname.1 b/man/pt_BR/domainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/pt_BR/domainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
diff --git a/man/pt_BR/hostname.1 b/man/pt_BR/hostname.1
deleted file mode 100644
index 5880fc8..0000000
--- a/man/pt_BR/hostname.1
+++ /dev/null
@@ -1,191 +0,0 @@
-.TH HOSTNAME 1 "28 de janeiro de 1996" "net-tools" "Manual do Programador Linux"
-
-.SH NOME
-hostname \- mostra ou configura o nome da máquina 
-.BR
-domainname \- mostra ou configura o nome do domínio NIS/YP
-.BR
-dnsdomainname \- mostra o nome do domínio DNS
-.BR
-nisdomainname \- mostra ou configura o nome do domínio NIS/YP
-.BR
-ypdomainname \- mostra ou configura o nome do domínio NIS/YP
-
-.SH SINOPSE
-.B hostname
-.RB [ \-v ]
-.RB [ \-a ]
-.RB [ \-\-alias ]
-.RB [ \-d ]
-.RB [ \-\-domain ]
-.RB [ \-f ]
-.RB [ \-\-fqdn ]
-.RB [ \-i ]
-.RB [ \-\-ip-address ]
-.RB [ \-\-long ]
-.RB [ \-s ]
-.RB [ \-\-short ]
-.RB [ \-y ]
-.RB [ \-\-yp ]
-.RB [ \-\-nis ]
-
-.PP
-.B hostname 
-.RB [ \-v ]
-.RB [ \-F\ arquivo ]
-.RB [ \-\-file\ arquivo ]
-.RB [ máquina ]
-
-.PP
-.B domainname
-.RB [ \-v ]
-.RB [ \-F\ filename ]
-.RB [ \-\-file\ nome_arquivo ]
-.RB [ nome ]
-
-.PP
-.B hostname
-.RB [ \-v ]
-.RB [ \-h ]
-.RB [ \-\-help ]
-.RB [ \-V ]
-.RB [ \-\-version ]
-
-.PP
-.B dnsdomainname
-.RB [ \-v ]
-.LP
-.B nisdomainname
-.RB [ \-v ]
-.LP
-.B ypdomainname
-.RB [ \-v ]
-
-.SH DESCRIÇÃO
-.B hostname
-é o programa usado para configurar ou mostrar o nome corrente da máquina
-ou o domínio do sistema. Este nome é usado por muitos dos programas de
-rede para identificar a máquina. O nome do domínio também é usado pelo
-NIS/YP.
-
-.SS "OBTER NOME"
-Quando chamado sem argumentos o programa mostra os nomes correntes:
-
-.LP
-.B hostname
-mostrará o nome do sistema conforme retornado pela função
-.BR gethostname (2).
-
-
-.LP
-.B "domainname, nisdomainname, ypdomainname"
-mostrará o nome do sistema conforme retornado pela função
-.BR getdomainname (2).
-Isto também é conhecido como o nome de domínio YP/NIS do sistema.
-
-.LP
-.B dnsdomainname
-mostrara a parte do domínio do FQDN (Nome de domínio completamente qualificado).
-O FQDN do sistema é retornado pelo comando
-.BR "hostname \-\-fqdn" .
-
-.SS "CONFIGURAR NOME"
-Quando chamado com um argumento ou com a opção
-.B \-\-file
-, o comando configura o nome da máquina ou do domínio NIS/YP.
-
-.LP
-Note que somente o super usuário pode mudar os nomes.
-
-.LP
-Nao é possível configurar o FQDN ou o nome do domínio DNS com o comando
-.B dnsdomainname 
-(veja 
-.B "O FQDN"
-abaixo).
-
-.LP
-O nome da máquina é normalmente configurado durante a inicialização do sistema,
-em
-.I /etc/rc.d/rc.inet1
-ou
-.I /etc/init.d/boot
-(normalmente lendo o conteúdo de um arquivo que contém o nome da máquina, ex.:
-.IR /etc/hostname ).
-
-.SS O FQDN
-Você não pode mudar o FQDN (conforme retornado por
-.BR "hostname \-\-fqdn" ) 
-ou o nome do domínio DNS (conforme retornado por
-.BR "dnsdomainname" ) 
-com este comando. O FQDN do sistema é o nome que o
-.BR resolver (3)
-retorna para o nome da máquina.
-
-.LP
-Tecnicamente: O FQDN é o nome retornado pelo
-.BR gethostbyname (2)
-para o nome retornado pelo
-.BR gethostname (2).
-O nome do domínio DNS é a parte após o primeiro ponto.
-.LP
-Portanto isto depende da configuração (normalmente em
-.IR /etc/host.conf )
-para que você possa mudá-lo. Normalmente (se o arquivo hosts for lido antes 
-do DNS ou NIS) você pode mudá-lo em
-.IR /etc/hosts .
-
-
-.SH OPÇÕES
-.TP
-.I "\-a, \-\-alias"
-Mostra o alias da máquina (se usado).
-.TP
-.I "\-d, \-\-domain"
-Mostra o nome do domínio DNS. Não use o comando
-.B domainname
-para obter o nome do domínio DNS porque ele mostrará o nome do domínio NIS e
-não o nome do domínio DNS. Use 
-.BR dnsdomainname .
-.TP
-.I "\-F, \-\-file arquivo"
-Leia o nome da máquina a partir do arquivo especificado. Comentários (linhas
-começando com um `#') são ignorados.
-.TP
-.I "\-f, \-\-fqdn, \-\-long"
-Mostra o FQDN (Nome de Domínio Completamente Qualificado). Um FQDN consiste de
-um nome curto de máquina e do nome do domínio DNS. A menos que você esteja
-usando bind ou NIS para resolução de nomes você pode mudar o FQDN e o nome
-do domínio DNS (que é parte do FQDN) no arquivo \fI/etc/hosts\fR.
-.TP
-.I "\-h, \-\-help"
-Mostra uma mensagem sobre como utilizar o comando e termina.
-.TP
-.I "\-i, \-\-ip-address"
-Mostra o(s) endereço(s) IP da máquina.
-.TP
-.I "\-s, \-\-short"
-Mostra o nome curto da máquina. É o nome da máquina até o primeiro ponto.
-.TP
-.I "\-V, \-\-version"
-Mostra informação de versão na saída padrão e termina com sucesso.
-.TP
-.I "\-v, \-\-verbose"
-Mostra uma saída detalhada do que está acontecendo.
-.TP
-.I "\-y, \-\-yp, \-\-nis"
-Mostra o nome do domínio NIS. Se um parâmetro for especificado (ou
-.B \-\-file arquivo
-) então o root também pode configurar um novo domínio NIS.
-.SH ARQUIVOS
-.B /etc/hosts
-.SH AUTOR
-Peter Tobias, <tobias@et-inf.fho-emden.de>
-.BR
-Bernd Eckenfels, <net-tools@lina.inka.de> (NIS e página man).
-.BR
-Arnaldo Carvalho de Melo, <acme@conectiva.com.br> Tradução para a língua
-portuguesa.
-.BR
-Jorge Luiz Godoy Filho, <jorge@bestway.com.br> Revisão.
-.BR
diff --git a/man/pt_BR/nisdomainname.1 b/man/pt_BR/nisdomainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/pt_BR/nisdomainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
diff --git a/man/pt_BR/ypdomainname.1 b/man/pt_BR/ypdomainname.1
deleted file mode 100644
index 1f45128..0000000
--- a/man/pt_BR/ypdomainname.1
+++ /dev/null
@@ -1 +0,0 @@
-.so man1/hostname.1
-- 
1.7.2.3

